frontends/ast/ast.cc

```diff
@@ -1469,6 +1471,7 @@ RTLIL::IdString AstModule::derive(RTLIL::Design *design, dict<RTLIL::IdString, R
 				log_error("No port with matching name found (%s) in %s. Stopping\n", log_id(intf.first), modname.c_str());
 			}
 		}
+		mod->fixup_ports();
```

Comment:
```suggestion
```

Reply:
Oup, forgot to remove that in 60405939943cd812e146b84848be8bc9307702db